dags/mag_and_friends.py

```diff
@@ -334,8 +336,16 @@ def create_dag():
                                 task_group=current_tasks
                             )
 
+                            copy_taxprofiler_results_from_cluster = cluster_manager.copyFromCluster(
```

Comment:
Are there ways to re-initiate transfer if the dag task failed or something? I had this happen before, and would like to find ways to better solve it.

Reply:
i havent yet seen cluster manager fail to copy files to remote and back for a reason other than a bug.. what is it you were seeing?

Reply:
dag task failed to start mostly, the error is "unable to access log file"

Reply:
that error doesnt seem specific to cluster manager or copying files or any particular task. ive looked up the error in airflow docs and source, and it seems like a large number of things could cause it, all to do w our setup/ install/ configuration. in our case, I suspect a concurrency issue w the postgres db, or similar, tbh.